pages/phases/getting-started/identifying-appropriate-positions.md

```diff
@@ -22,10 +22,14 @@ Your position description (PD) should reflect adequate specialized experience to
 
 ### Identify Billets
 
-To be efficient with SME time, we recommend having at least three and ideally five or more opening/billets to fill in your hiring action. 
+We recommend having at least three and ideally five or more opening/billets to fill in your hiring action. This ensures that the time SMEs spend effectively assessing applicants is leveraged through multiple hires. It also saves time for applicants and HR: if you're able to keep a certiciate open for a year and make 5+ hires off of one hiring action there is less of a need to have applicants apply to many different one-off vacancies.  Ideas if you do not have at least 3-5 openings:
+
+- Try a shared hiring agency with another office or agency with openings for the same position. This way there are more SMEs involved therefore decreasing the overall time burden on one agency's SMEs. USDS recently completed a SME-QA hiring action with 20 SMEs from 9 agencies hiring Customer Experience Specialists. OPM was hired as the HR Specialist and agencies were able to share the certificate the same day. If two agencies want to partner on their own, they are allowed to share the certificate as long as they have applicants opt-in when they apply and have a merit promotion announcement open for 10 days before the second agency makes selections. 
+
+- If you have 5+ openings for similiar roles that share the same core competetencise but have slightly different specialized experience required for each role, you can have the SMEs assess resumes and conduct the first interview to ensure the core competenies are met. The second interview can be dedicated to seeing which applicants meet the additional competency for each slightly different role. Separate certificates can be issued at the end of the hiring action for each sub-role within the announcement. The first two SME-QA pilots at HHS and DOI deployed this strategy since the participating offices all had the same core requirements but slightly different specialized experience for each sub-role.
```

Comment:
```suggestion
- If you have 5+ openings for similar roles that share the same core competencies but have slightly different specialized experience required for each role, you can have the SMEs assess resumes and conduct the first interview to ensure the core competencies are met. The second interview can be dedicated to seeing which applicants meet the additional competency for each slightly different role. Separate certificates can be issued at the end of the hiring action for each sub-role within the announcement. The first two SME-QA pilots at HHS and DOI deployed this strategy since the participating offices all had the same core requirements but slightly different specialized experience for each sub-role.
```

Reply:
Do you want to mention parentheticals elsewhere and then link to that mention? It seems like "how to do parentheticals" is different than identifying positions.